src/dymaptic.GeoBlazor.Core/Components/Views/MapView.razor.cs

```diff
@@ -1414,9 +1414,24 @@ public async Task AddGraphics(IEnumerable<Graphic> graphics, CancellationToken c
     }
 
 #if NET7_0_OR_GREATER
+    internal void AddGraphicsSyncInterop(byte[] graphics, string id, string? layerId = null)
+    {
+        if (ProJsViewModule is not null)
+        {
+            AddGraphicsProSyncInterop(graphics, id, layerId);
```

Comment:
I assume the pro code PR will have this method adjustment, since the `AddGraphicsCoreSyncInterop` is the only method in Core's `arcGISJsInterop.ts`?

Reply:
Yes, the method didn't actually exist at all in `arcGisPro.ts` yet, it is added in that PR.